### DISCUSSION TITLE
Important Notice: Upcoming Plugin Preview Service Deprecation

### DISCUSSION THREAD

**Discussion by dgmccart** [chanzuckerberg/napari-hub#1216] [Announcements]
Since 2021, the napari hub plugin preview service allowed developers to see how their plugin page would look every time they opened a new pull request, or updated an existing one, and provide guidance on what to include in their plugin page. However, very few developers utilize this feature, and the broader napari community is still in discussion about what elements to recommend to include in plugin pages. As a result, we have decided to remove the plugin preview service on September 1, 2023.  Please reach out to me ([dmccarthy@chanzuckerberg.com](mailto:dmccarthy@chanzuckerberg.com)) if you have any questions or concerns regarding this decision by Friday, August 18.

**Comment by MMathisLab** [comment]
Ah very sorry to see this go -- we used this feature extensively in napari-deeplabcut !
